examples/hybrid/sphere/deformation_flow.jl

```diff
@@ -266,7 +263,8 @@ end
 
 # Set up RHS function
 ystar = copy(y0)
-parameters = (τ = τ, coords = coords, face_coords = face_coords)
+parameters =
+    (; τ = τ, coords = coords, face_coords = face_coords, ystar = ystar)
```

Comment:
```suggestion
parameters = (; τ, coords, face_coords, ystar)
```
Just to note, this syntax works, too